api/envoy/api/v2/listener/listener.proto

```diff
@@ -93,6 +94,18 @@ message FilterChainMatch {
   // [#not-implemented-hide:]
   google.protobuf.UInt32Value suffix_len = 5;
 
+  enum ConnectionSourceType {
+    // Any connection origin matches
+    ANY = 0;
+    // Match a connection originating from the same host
+    LOCAL = 1;
+    // Match a connetion originating from a different host
+    EXTERNAL = 2;
+  }
+
+  // Specifies the connection source IP match type. Can be any, same or distinct network
```

Comment:
Nit: comment still refers to same and distinct and not local and external.

Reply:
Done. Thanks.